manifests/package/apt.pp

```diff
@@ -1,14 +1,12 @@
 # package/apt.pp (private class)
-# Uses the puppetlabs-apt module to manage apt package manager proxies
-# https://forge.puppetlabs.com/puppetlabs/apt
+# Writes a conf.d file to make Apt use the proxy
 class httpproxy::package::apt {
-
-  class { '::apt':
-    proxy => {
-      'ensure' => $httpproxy::packagemanager::ensure,
-      'host'   => $httpproxy::http_proxy,
-      'port'   => $httpproxy::http_proxy_port,
-    },
+  file { 'apt_via_proxy':
+    ensure  => $httpproxy::packagemanager::ensure,
+    path    => '/etc/apt/apt.conf.d/05proxy',
```

Comment:
https://github.com/puppetlabs/puppetlabs-apt/blob/v7.1.0/manifests/init.pp#L194

I suggest /etc/apt/apt.conf.d/01proxy so that we don't end up with 2 files when converting and maintain backwards compatability